lua/weapons/gmod_tool/stools/wire_magnet.lua

```diff
@@ -46,7 +46,7 @@ function TOOL:LeftClick( trace )
 	
 	local leng 		= tonumber(self:GetClientNumber( "leng" ))
 	local strength	 	= tonumber(self:GetClientNumber("streng" ))
-	local propfilter	 	= tostring(self.ClientConVar[ "propfilter" ])
+	local propfilter	 	= string.lower( self:GetClientInfo( "propfilter" ) ) --Fixed. local propfilter = tostring(self.ClientConVar[ "propfilter" ]) Does not work.

```

Comment:
This comment here is unnecessary - the reader doesn't need to know what previous line of code was there that didn't work.